tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "target": "esnext",
+    "module": "esnext",
```

Comment:
What would happen if we don't add this?

Reply:
From what I can gather by commenting it out, I _think_ it was added so that we could do dynamic imports for the devtools tests. We want to dynamically import the tools to avoid needing to clean up connections from previous tests. @pavlobu would know for sure.

Reply:
Correct @cwstra , that was done for that. To use dynamic imports inside tests.
Thanks!

Reply:
Thanks for clarification!

@pavlobu Can you elaborate why we need dynamic imports in tests?